frontend/utilities/constants.ts

```diff
@@ -17,11 +17,11 @@ export const DEFAULT_POLICIES = [
   {
     key: 1,
     query: `SELECT 1 FROM disk_encryption WHERE user_uuid IS NOT "" AND filevault_status = 'on' LIMIT 1`,
-    name: "Is FileVault enabled on macOS devices?",
+    name: "Full disk encryption enabled (macOS)",
     description:
-      "Checks to make sure that the Filevault feature is enabled on macOS devices.",
+      "Checks to make sure that full disk encryption is enabled on macOS devices.",
```

Comment:
In the description and resolution, I think we should still mention Filevault. Maybe:

Checks to make sure that full disk encryption (Filevault) is enabled on macOS devices.